Example/Program.cs

```diff
@@ -0,0 +1,36 @@
+using Example.Extensions;
+using Flagsmith;
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.RegisterFlagsmithClientAsSingleton(builder.Configuration);
+var app = builder.Build();
+
+app.MapGet("/", async (FlagsmithClient flagsmithClient) =>
```

Comment:
As discussed on the other PR, let's bring this closer to the python example so it looks more like a real world app. There should be a readme in this example directory too about how to run the example application.

Reply:
It's definitely closer, yes, but the python example uses an HTML template [here](https://github.com/Flagsmith/flagsmith-python-client/blob/release/3.0.0/example/templates/home.html).Is there an easy way to include a template like that in the dotnet example app?